styleguide/_themes/derek/scss/patterns/announcements.scss

```diff
@@ -0,0 +1,88 @@
+.announcement-container {
+  background-color: $white;
+  border-bottom: 1px solid $light-gray;
+  border-right: 1px solid $light-gray;
+  border-top: 1px solid $light-gray;
+  margin-bottom: 10px;
+  overflow: hidden;
+  padding: 10px;
+  position: relative;
+  width: $full-width;
+}
+
+.announcement-icon {
+  @include make-md-column(2);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 60px 60px;
+  min-height: 100px;
+  opacity: .7;
+  padding: 40px;
+  text-align: center;
+}
+
+.announcement-message {
+  @include make-md-column(10);
+  min-height: 100px;
+  padding-top: 12px;
+}
+
+.announcement-close {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/icons/small-close.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 10px;
+  position: absolute;
+  right: 8px;
+  top: 8px;
+  width: 10px;
+  z-index: 5;
+}
+
+//Offer announcements
+.announcement-offer {
+  border-left: 8px solid $mediumteal;
+
+  .announcement-icon {
+    background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/icons/rsweb-icon-piggybank.svg');
+  }
+}
+
+//Warning announcements
+.announcement-warning {
+  border-left: 8px solid $mustard;
+
+  .announcement-icon {
+    background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/icons/rsweb-icon-warning.svg');
+  }
+}
+
+//Danger announcements
+.announcement-danger {
+  border-left: 8px solid $rackspace-red;
+
+  .announcement-icon {
+    background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/icons/rsweb-icon-danger.svg');
+  }
+}
+
+//Success announcements
+.announcement-success {
+  border-left: 8px solid $bright-green;
```

Comment:
can you use signup green for this one. I want to try to use bright-green. the only place its used is business resources and I'd rather phase it out if possible.